types/RootStackParamList.ts

```diff
@@ -10,10 +10,17 @@ export type RootStackParamList = {
 
   Home: undefined;
   Camera: undefined;
+
+  SetUsername: undefined;
+  MaximizeScreen: undefined;
+  FriendsScreen: undefined;
+  CreateChallengeScreen: {
+
   Maximize: {
```

Comment:
Maximize is the name in the Stack to navigate to maximize_screen you must write "Maximize"